docs/_docs/editor/setup.md

```diff
@@ -29,12 +29,11 @@ $ apm install nuclide
 or within Atom itself:
 
 1. Open Atom.
-2. Choose `Atom | Preferences` (on Linux this will be
-  `Edit | Preferences`) to bring up the `Settings` pane.
-3. In the `Settings` pane, click on `Install`.
-4. In the search box, type *Nuclide* and press the *Enter* key.
+2. Choose `Atom | Preferences` (on Linux this will be `Edit | Preferences`) to bring up the **Settings** tab.
+3. In the **Settings** tab, click on **Install**.
+4. In the search box, type "Nuclide" and press the `Enter` key.
 5. The package you are looking for is called `nuclide` and should be version `0.111.0` or greater.
```

Comment:
This reference to the version number can be removed. When you look for "nuclide", Nuclide is the first result. And the rest are unambiguously not Nuclide.